core/src/main/scala/kafka/log/LogConfig.scala

```diff
@@ -137,9 +137,9 @@ object LogConfig {
     "standard compression codecs ('gzip', 'snappy', lz4). It additionally accepts 'uncompressed' which is equivalent to " +
     "no compression; and 'producer' which means retain the original compression codec set by the producer."
   val PreAllocateEnableDoc ="Should pre allocate file when create new segment?"
-  val MessageFormatVersionDoc = KafkaConfig.MessageFormatVersionDoc
-  val MessageTimestampTypeDoc = KafkaConfig.MessageTimestampTypeDoc
-  val MessageTimestampDifferenceMaxMsDoc = KafkaConfig.MessageTimestampDifferenceMaxMsDoc
+  val MessageFormatVersionDoc = KafkaConfig.LogMessageFormatVersionDoc
```

Comment:
I am not sure I understand. I didn't rename the LogConfig variables, I renamed the ones in KafkaConfig.